### ISSUE TITLE
As a developer, I need our NextGenXMS integration unit tested

### ISSUE THREAD

**Issue by andrew-jameson** [raft-tech/TANF-app#1558]
**Description:**
_Provide a brief background and justification for this issue_
For greater throughput, we have decided to make test writing a separate ticket for this integration. 

**Acceptance Criteria:** 
_Create a list of functional outcomes that must be achieved to complete this issue_

- [ ] Tests cover new functions and classes as set out in #1154.
- [ ] Tests adhere to newer format of #1540
- [ ] Authentication backend adheres to Codecov minimum coverages
- [ ] Testing Checklist has been run and all tests pass
- [ ] README is updated, if necessary

**Tasks:**
_Create a list of granular, specific work items that must be completed to deliver the desired outcomes of this issue_ 

- [ ] Write new unit tests for NextGenXMS integration work
- [ ] Run Testing Checklist and confirm all tests pass

**Notes:**
_Add additional useful information, such as related issues and functionality that isn't covered by this specific issue, and other considerations that will be helpful for anyone reading this_ 

**Supporting Documentation:**
_Please include any relevant log snippets/files/screen shots_

**Open Questions:**
_Please include any questions or decisions that must be made before beginning work or to confidently call this issue complete_

**Comment by valcollignon** [comment]
Dependent on #1540. Format must be established before this ticket can be completed. Per backlog refinement 1.25.22

**Comment by stevenino** [comment]
Required for v1.0

**Comment by stevenino** [comment]
Will not integrate with NexGen XMS
